src/main/java/org/candlepin/subscriptions/inventory/db/InventoryRepository.java

```diff
@@ -20,8 +20,8 @@
  */
 package org.candlepin.subscriptions.inventory.db;
 
-import static org.hibernate.jpa.QueryHints.HINT_FETCH_SIZE;
-import static org.hibernate.jpa.QueryHints.HINT_READONLY;
+import static org.hibernate.jpa.HibernateHints.HINT_FETCH_SIZE;
```

Comment:
In the QueryHints documentation they have proposed to use AvailableHints. I suggest to use that however AvailableHints use HibernateHints internally so, either ways it is fine.
Reference: https://docs.jboss.org/hibernate/orm/6.0/javadocs/org/hibernate/annotations/QueryHints.html